pkg/server/sandbox_run.go

```diff
@@ -559,6 +562,16 @@ func (c *criService) setupPod(id string, path string, config *runtime.PodSandbox
 	return "", result, errors.Errorf("failed to find network info for sandbox %q", id)
 }
 
+// toCNIPortMappings converts CRI annotations to CNI bandwidth.
+func toCNIBandWidth(annotations map[string]string) cni.BandWidth {
```

Comment:
We could reuse the functionality from: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/bandwidth/utils.go

what do you mean?

Reply:
sg